src/_common/list-group/selector/selector.vue

```diff
@@ -0,0 +1,32 @@
+<template>
+	<div class="list-group">
```

Comment:
Doesn't center correctly with long content
![image](https://user-images.githubusercontent.com/996947/68133724-9892a480-ff29-11e9-9919-4f1070e22d07.png)

Reply:
Current issue, not something this PR introduced.